homeassistant/generated/zeroconf.py

```diff
@@ -205,6 +205,9 @@
         }
     ],
     "_kizbox._tcp.local.": [
+        {
+            "domain": "overkiz"
```

Comment:
Would it make sense to add the name as well? (see Somfy zeroconf). This will perhaps be more strict?

```
            "name": "gateway*"
```